playbooks/ORCHESTRATION.md

```diff
@@ -10,18 +10,7 @@ This playbook describes how to orchestrate a root signing event.
 
 3. Double-check the configured [role expirations](https://github.com/sigstore/root-signing/blob/e3f1fe5e487984f525afc81ac77fa5ce39737d0f/cmd/tuf/app/init.go#L28).
 
-4. Create (or ask a root-signing maintainer) to create a new upstream branch named by the ceremony date, for example, `ceremony/YYYY-MM-DD`.
-
-5. Set any environment variables, including the repository and online signer references (details [here](#key-configuration)).
-
-| Variable      | Description | Example |
-| ----------- | ----------- | ----------- |
-| GITHUB_USER      | The GitHub user, used to create PRs and commit messages       | asraa       |
```

Comment:
to confirm, you dont need to set this anymore?

Reply:
oh nvm you set this later

Reply:
Yes! You only need to set it for local testing (so not for any of the workflow creation)